lib/input.js

```diff
@@ -0,0 +1,22 @@
+const minimist = require('minimist');
+
+const parse = (arr) => {
+  const obj = minimist(arr);
+  delete obj._;
+
+  const [[type, payload]] = Object.entries(obj);
+
+  return {
+    type,
+    payload
+  };
+};
+
+const valid = action => {
+  return action.type === 'add' && action.payload;
+};
+
+module.exports = { 
+  parse,
+  valid
+};
```

Comment:
Files should end with a newline. you can configure vs code to add it for you automatically